src/popup.js

```diff
@@ -164,7 +164,11 @@ chrome.runtime.onMessage.addListener((request) => {
         hideLoading();
       });
   } else if (request.prsUpdated) {
-    updatePRs();
+    updatePRs()
+      .then(()=>{
```

Comment:
Just to give you a taste of the fully nit picking I'd usually do for the fat arrows you should have a space on either side for readability. 

```
     .then(() => { 
```